templates/terraform/examples/bigquery_reservation_basic.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_bigquery_reservation" "<%= ctx[:primary_resource_id] %>" {
+	provider       = google-beta
+	name           = "<%= ctx[:vars]['name'] %>"
+	location       = "asia-northeast1"
+	// Set to 0 for testing purposes
+	// In reality this would be larger than zero
+	slot_capacity  = 0
```

Comment:
If we don't need a hard-to-get quota, let's use a positive # (probably 1-3) since this shows up in the docs.

Reply:
Unfortunately we would need a pretty expensive minimum commitment to be able to use any positive values here: https://cloud.google.com/bigquery/pricing#flat_rate_pricing